@LinProp/LinProp.m

```diff
@@ -279,17 +279,17 @@ function display(obj)
             else
                 if obj.IsComplex
                     sreal = ['(' num2str(abs(get_value(real(obj))), df) ...
-                             ' � ' num2str(get_stdunc(real(obj)), df) ')'];       
+                             ' ± ' num2str(get_stdunc(real(obj)), df) ')'];       
```

Comment:
Did you changed the files to unicode?

Reply:
Oh no! I used GitHub's built-in editor to clear a conflict when merging the 'fix-subsasgn-with-empty-indices' branch. That might have caused the change... Matlab 2021a (which I mainly use) uspports unicode files so I usually do not see changes in the encoding.